web_app_drawer/static/src/js/web_app_drawer.js

```diff
@@ -0,0 +1,183 @@
+            var $clickZones = $('.openerp_webclient_container, ' +
+                                'a.oe_menu_leaf, ' +
+                                'a.oe_menu_toggler'
+                                );
+            $clickZones.click($.proxy(this.handleClickZones, this));
+            this.$el.one('drawer.opened', $.proxy(this.onDrawerOpen, this));
+            // Core bus keypress doesn't seem to send arrow keys
+            core.bus.on('keydown', this, this.handleNavKeys);
+        },
+        
+        // It provides handlers to hide drawer when "unfocused"
+        handleClickZones: function() {
+            this.$el.drawer('close');
+            $('.oe_secondary_menus_container')
+                .parent()
+                .collapse('hide');
+        },
+        
+        // It provides keyboard shortcuts for app drawer nav
+        handleNavKeys: function(e) {
+            if (!this.isOpen){
+                return;
+            }
+            if (Object.keys(this.directionCodes).indexOf(e.keyCode.toString()) !== -1) {
+                var $link = this.findAdjacentAppLink(
+                    this.$selectedAppLink,
+                    this.directionCodes[e.keyCode][0],
+                    this.directionCodes[e.keyCode][1]
+                );
+                this.selectAppLink($link);
+            }
+        },
+        
+        onDrawerClose: function() {
+            this.isOpen = false;
+            this.$selectedAppLink = $(false);
+            core.bus.trigger('drawer.closed');
+            this.$el.one('drawer.opened', $.proxy(this.onDrawerOpen, this));
+        },
+        
+        // Generate x and y coords for app icons
+        onDrawerOpen: function() {
+            var self = this;
+            this.isOpen = true;
+            this.$appLinks = $('.css-app-drawer-icon-app').parent();
+            this.selectAppLink($(this.$appLinks[0]));
+            this.linkMap = {};  // Key by row, col
+            var x = 0,
+                y = -1,
+                lastTop = -9999,
+                newTop = null,
+                $lastVal = null;
+            _.each(this.$appLinks, function(val) {
+                $lastVal = $(val);
+                newTop = $lastVal.offset().top;
+                if (lastTop < newTop) {
+                    y ++;
+                    x = 0;
+                    self.linkMap[y] = {};
+                }
+                self.linkMap[y][x] = $lastVal;
+                $lastVal.data('pos-x', x).data('pos-y', y);
+                x ++;
+                lastTop = newTop;
+            });
+            this.$appLinks.on('mouseenter', $.proxy(this.selectAppLinkEvent, this));
+            this.$el.one('drawer.closed', $.proxy(this.onDrawerClose, this));
+            core.bus.trigger('drawer.opened');
+        },
+        
+        // It provides a proxy method allowing for app link select w/ events
+        selectAppLinkEvent: function(event) {
+            this.selectAppLink($(event.target));
```

Comment:
Please use `event.currentTarget` to avoid this:

![captura de pantalla de 2016-08-23 18-13-12](https://cloud.githubusercontent.com/assets/973709/17899926/58d6cad4-695d-11e6-9200-4ccdb9674f15.png)

Docs:

- http://api.jquery.com/event.target/
- http://api.jquery.com/event.currentTarget/